### PR TITLE
metadata zip

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[[RESPECT] 9.5.1/9.5.2 Metadaten exportieren](https://trello.com/c/sPhm7OyK/922-respect-951-952-metadaten-exportieren)

File: examples/ndvi_ports.ipynb [hunk 1]
```diff
@@ -440,6 +442,39 @@
...
+     "text": [
+      "File Name                                             Modified             Size\n",
+      "workflow.json                                  2022-07-21 09:24:18          772\n",
+      "metadata.json                                  2022-07-21 09:24:18         1074\n",
+      "citation.json                                  2022-07-21 09:24:18          794\n"
+     ]
+    }
+   ],
+   "source": [
+    "zip_content = io.BytesIO() # normally, a file path\n",
```
Comment (michaelmattig) [inline examples/ndvi_ports.ipynb:470]: maybe replace "normally" with "alternatively"

Comment (ChristianBeilschmidt) [reply examples/ndvi_ports.ipynb]: [3a66a6c](https://github.com/geo-engine/geoengine-python/pull/65/commits/3a66a6ca3268c498dbdc29466219ba8136e5d5e3)

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**